app/helpers/casino/sessions_helper.rb

```diff
@@ -23,6 +23,10 @@ def current_user
     tgt.user
   end
 
+  def current_authenticator_context
+    CASino.config.authenticator_context_builder.call(params, request)
```

Comment:
I was more thinking about: How is the config option set? Options are currently loaded from the `cas.yml` which in this case will not work (since it has to be a `Proc`). Is the suggested approach to set this specific option in an initializer?